lib/go_script/command_group.rb

```diff
@@ -66,6 +66,7 @@ def add_group(group_symbol, description, path, lineno)
 
       def command_defined?(command)
         groups.values.each { |g| return true if g.include_command? command }
```

Comment:
+1 to Mr. Creosote reference BTW

On Fri, Aug 28, 2015 at 10:48 AM, Mike Bland <notifications@github.com>
wrote:

> In lib/go_script/command_group.rb
> <https://github.com/18F/go_script/pull/7#discussion_r38207328>:
>
> > @@ -66,6 +66,7 @@ def add_group(group_symbol, description, path, lineno)
> >
> >        def command_defined?(command)
> >          groups.values.each { |g| return true if g.include_command? command }
>
> D'oh! Great catch. Done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/go_script/pull/7/files#r38207328>.
>



-- 
Alison Rowland
Technical Lead, https://fbopen.gsa.gov | 18F
202-317-0124